README.adoc

```diff
@@ -2548,6 +2548,24 @@ some_method('w', 'x', 'y') # => 'y, 2, w, x'
 some_method('w', 'x', 'y', 'z') # => 'y, z, w, x'
 ----
 
+=== Optional Keyword Arguments [[optional-keywordarguments]]
```

Comment:
I think the name of the section is a bit misleading, as the its actual point is to sort the required and the optional keyword arguments. Something like `Keyword Arguments Order` sounds better to me.

Reply:
Sure, changed.